doc/produs_unterlagen/solution/controls.md

```diff
@@ -0,0 +1,47 @@
+## Solutions for all Control Levels and the Berry Level
+
+### Solution for `MyPlayerController`
+
+```java
+protected void processKey(String key) {
+    if (key.equals("W")) move(0, 5);
+    if (key.equals("S")) move(0, -5);
+    if (key.equals("A")) move(-5, 0);
+    if (key.equals("D")) move(5, 0);
```

Comment:
damit ist schräg laufen nicht möglich, oder?
Ich setze ja immer den anderen Achsen-Wert auf 0.

Eig müsste das sowas sein wie `if (key.equals("W")) move(DONT_CHANGE, 5);`

jetzt frag mich grade nur nicht wie das gut für die kidz machbar ist

Reply:
edit ah i c unten in move